src/shared/ui/components/select/select.component.tsx

```diff
@@ -47,7 +47,7 @@ export const Select = <T,>({
           <DropdownMenu.Content sideOffset={2} asChild>
             <SelectOptionContainer
               as="div"
-              className="w-[var(--radix-popper-anchor-width)] text-black"
+              className="max-h-64 w-[var(--radix-popper-anchor-width)] overflow-y-auto text-black"
```

Comment:
Also for future cases it might be worth to add extra prop to this component which would allow consumers to change the max-height of options, I can imagine we can face a scenario where options should be taller/smaller etc.

You would need to add new prop to this component `optionsContainerClassName?: string` and merge these classes via our utility:
`className={classes('max-h-64 w-[var(--radix-popper-anchor-width)] overflow-y-auto text-black', optionsContainerClassName)}`

But we can also add it if we face this situation, that's minor and I'll leave it up to you if you want to add it or not.

Reply:
Good idea, optionsContainerClassName prop added